layout/_third-party/math/mathjax.swig

```diff
@@ -1,18 +1,29 @@
 <script type="text/x-mathjax-config">
+  {% if theme.math.mathjax.mhchem %}
+    MathJax.Ajax.config.path['mhchem'] = '{{ theme.math.mathjax.mhchem }}';
+  {% endif %}
+
   MathJax.Hub.Config({
     tex2jax: {
-      inlineMath: [ ['$','$'], ["\\(","\\)"]  ],
+      inlineMath: [ ['$','$'], ["\\(","\\)"] ],
       processEscapes: true,
       skipTags: ['script', 'noscript', 'style', 'textarea', 'pre', 'code']
     },
-    TeX: {equationNumbers: { autoNumber: "AMS" }}
+    TeX: {
+      {% if theme.math.mathjax.mhchem %}
+        extensions: ['[mhchem]/mhchem.js'],
```

Comment:
Oh, i see. Maybe try to add in CDN's settings something like `//cdn.jsdelivr.net/npm/mathjax-mhchem@3/mhchem.min.js` and stay here only `extensions: ['[mhchem]']`?

Reply:
You're right, `//cdn.jsdelivr.net/npm/mathjax-mhchem@3` is confusing... I'll fix it

Reply:
Mb because `MathJax.Ajax.config.path` exists above.
Ok, don't stay it as is. Mb later we can optimize it. :1st_place_medal: